Microservices/event-service/src/test/java/app/sportahub/eventservice/EventServiceApplicationTests.java

```diff
@@ -0,0 +1,15 @@
+package app.sportahub.eventservice;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Import;
+
+@Import(TestcontainersConfiguration.class)
```

Comment:
Can you remove this since it loads the TestcontainersConfiguration.class and the current class is for unit tests whereas TestcontainersConfiguration.class is for integration tests. This line also causes the contextLoads() to fail.

Reply:
Done